web/business-registry-dashboard/tests/mocks/i18n.ts

```diff
@@ -1,27 +1,29 @@
 import { createI18n } from 'vue-i18n'
-import en from '~~/app/locales/en-CA'
-import fr from '~~/app/locales/fr-CA'
+import en from '~~/i18n/locales/en-CA'
+import fr from '~~/i18n/locales/fr-CA'
 
 export const enI18n = createI18n({
   legacy: false,
   locale: 'en-CA',
   messages: {
-    'en-CA': en
+    'en-CA': en,
+    'fr-CA': fr
   }
 })
 
 export const frI18n = createI18n({
   legacy: false,
   locale: 'fr-CA',
   messages: {
-    'fr-CA': fr
+    'fr-CA': fr,
+    'en-CA': en
   }
 })
 
-export const randomI18n = createI18n({
-  legacy: false,
-  locale: 'ja',
-  messages: {
-    ja: en
-  }
-})
+// export const randomI18n = createI18n({
```

Comment:
This has an error and doesn't seem to be used, so I just commented it out for now.